buildlib/tools/builds.sh

```diff
@@ -407,6 +407,14 @@ build_static() {
 
 	${WORKSPACE}/buildlib/tools/check_tls.sh $EXTRA_TLS
 
+	for tls in tcp $RUN_TLS; do
+		echo UCX_TLS=$tls
+		UCX_TLS=$tls ./ucp_hello_world_static &
```

Comment:
1. can we derive the TCP port number from agent id, like in test_jenkins.sh
since we share net namespace with host, it can create collisions

2. we should move the static test from builds.sh to a separate script since it doesn't just "build" now, it also runs tests. can be in this or another PR.

Reply:
> * can we derive the TCP port number from agent id, like in test_jenkins.sh
>   since we share net namespace with host, it can create collisions

should not be issue because containers are launched in separate network namespaces and run in single thread

> * we should move the static test from builds.sh to a separate script since it doesn't just "build" now, it also runs tests. can be in this or another PR.

will do later - need deeper changes (different infrastructure for variables and container initialization)

Reply:
> should not be issue because containers are launched in separate network namespaces and run in single thread

Actually, we run them with "-net host" in Azure

Reply:
> Actually, we run them with "-net host" in Azure

as I can see it is launched in own network namespace:
```/scrap/bin/docker create --label 8f00dc --network vsts_network_e782202237e342969d72245c7a75d492 -v /hpc/local:/hpc/local ... ```

@avildema any ideas?

Reply:
`/scrap/bin/docker` is wrapper to use host net

Reply:
hmmm, does it just removes --network argument?
but ok, added custom server port

Reply:
you should add to command like --het host 
like this https://github.com/openucx/ucx/blob/master/buildlib/pr/main.yml#L16

Reply:
> you should add to command line `--net=host `

we need own network namespace for docker containers

Reply:
> we need own network namespace for docker containers

in general yes but it had some issues with TCP tests using bridge netorks (though i think it should be resolved now)
in any case, to be on safe side, IMO better generate TCP port num from agent id in this PR

Reply:
yep, done.
under testing...